### PR TITLE
Setup the web server backend and install basic security so we don't get hacked instantly

### PR DESCRIPTION
**Pull request by MartinBruun**

### CODE REVIEW

**Comment by MartinBruun** [comment]
The pandas, numpy, scikit-learn and similar packages have been uncommented in requirements.txt, due to them breaking the debian image when run on the raspbian system (which makes the tests involving MI fail)

There should be found a solution for that, before this gets merged in. It probably involves receiving and pre-compiling the source code somehow.

File: washee_web/washee_web/settings.py [hunk 1]
```diff
@@ -26,10 +26,19 @@
...
+
+# SECURITY WARNING! Controls all session based security
+SESSION_COOKIE_SECURE = False if os.environ.get("DJANGO_SESSION_COOKIE",default=1) == 0 else True
+
 # Create Fixture files that can load a whole database
-FIXTURE_DIRS = ["/washee_web"]
+FIXTURE_DIRS = ["/location"]
 
-ALLOWED_HOSTS = os.environ.get("DJANGO_ALLOWED_HOSTS").split(" ")
+ALLOWED_HOSTS = ["*"]#os.environ.get("DJANGO_ALLOWED_HOSTS").split(" ")
```
Comment (MartinBruun) [inline washee_web/washee_web/settings.py:41]: Shit, i must change this!

File: README.md [hunk 2]
```diff
@@ -61,7 +61,9 @@ https://raspberrytips.com/security-tips-raspberry-pi/
 Also Update the Django security
 https://opensource.com/article/18/1/10-tips-making-django-admin-more-secure
 
-
+How to setup https (things followed now)
```
Comment (MartinBruun) [inline README.md:64]: Also need better docs

**Review by MartinBruun — COMMENTED** [review]
Few Reminders to myself. Please review anyway when you have the time :-)

EDIT: Solved, it is now in final state

**Review by mkju19 — APPROVED** [review]

**Pull request merged**